entity-client-api/src/main/java/org/terracotta/entity/InvocationCallback.java

```diff
@@ -0,0 +1,45 @@
+package org.terracotta.entity;
+
+/**
+ * <p>The acknowledgement configurations can be complex so a high-level description of them and their meanings is
+ * provided here, with greater detail in the relevant methods, below:</p>
+ * <ul>
+ *  <li><b>SENT</b> means that a message has become "in-flight", meaning that it will be re-sent in the event of a
+ *  server restart or fail-over to passive. Note that this message is not generally useful to applications</li>
+ *  <li><b>RECEIVED</b> if requested, will only return from invoke() once the global order of this message (relative to
+ *  others depending on RECEIVED) has been determined for the entire stripe. This means that every server in the stripe
+ *  has received the message, and agreed upon the order. Note that RECEIVED is the only optional ack (the others always
+ *  go over the wire, but only optionally influence client-side waiting). This should be requested in cases where
+ *  knowing the global order of messages (preserved for restart or fail-over) is important</li>
+ *  <li><b>COMPLETED</b> is observed by the client when the active has finished running the invoke and returned the
+ *  result. Calling get() on the returned InvokeFuture will, by default, wait until RETIRED but the InvocationBuilder
```

Comment:
This sentence doesn't make sense in an async context right?

Reply:
Will fix.